.env

```diff
@@ -65,6 +66,10 @@ VOTE_MAP_INSTRUCTIONS=To vote write the map number in the chat
 # changes on your server. If you don't want to receive that info set it to 1
 SILENT_MAP_RECORDER=
 
+# If you would like to only allow players with a certain numbers of hours played
+# Requires STEAM_API_KEY to be set
+MIN_HLL_PLAYTIME_HOURS=
```

Comment:
I would suggest using https://github.com/MarechJ/hll_rcon_tool/blob/master/rcon/config.py#L11  that grab the config from `confg/config.default.py` (or the user override in `config/config.yml`

Because the `.env` causes conflict everytime user pull the repo so I'm trying not to use  it anymore.
As users are not supposed to change the config.default they don''t have this problem anymore.